typings/PickerIOS.d.ts

```diff
@@ -0,0 +1,27 @@
+import * as React from "react"
+import { TextStyle, StyleProp, ViewProps } from 'react-native'
+import { ItemValue } from "./Picker"
+
+export interface PickerIOSItemProps {
+	label?: string;
+	value?: number | string;
+	color?: string;
+	testID?: string;
+}
+
+class PickerIOSItem extends React.Component<PickerIOSItemProps, {}> {}
```

Comment:
Reflecting changes from https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41502

This line causes TypeScript users to be able to import `PickerItem` which isn't the component's name and may lead to confusion.
```suggestion
```